docs/components/beacon/index.html

```diff
@@ -0,0 +1,117 @@
+---
+title: Beacon
+minver: 0.23.0
+also:
+  elements/hx-beacon: <hx-beacon>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    A {{page.title}} is an animation that attracts a user's attention
+    to a new feature or advanced functionality in a control panel.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="single-beacon">Beacon</h2>
+      <p>
+        A beacon will emit a <code>dismiss</code> event when a user clicks it.
+        Additionally, it will be removed from the DOM.
+      </p>
+    </header>
+
+    <div class="example" id="vue-beaconDemo" v-cloak>
+      <div>
+        <hx-beacon
+          @dismiss="onEvent"
+        >
+        </hx-beacon>
+      </div>
+
+      <footer>
+        <pre><code v-text="snippet"></code></pre>
+     </footer>
+    </div>
+  </section>
+
+
+  <section>
+    <header>
+      <h2 id="basic-beacon-inline-styles">Basic Beacon with Inline Styles</h2>
+    </header>
+    <p>
+      This is an example implementation of <code>&lt;hx-beacon&gt;</code>.
+      We leave it to HelixUI implementors to decide what works best for their
+      applications.
+    </p>
+    <p>
+      <code>
+        NOTE: You may have to use <b>inline styles</b> to adjust
+        the <code>&lt;hx-beacon&gt;</code> location.
+      </code>
+    </p>
+    <div class="example" id="vue-beacon-inline-styles" v-cloak>
+      <header>
+        <form class="beta-hxForm">
+          <fieldset>
+            <!-- <legend class="beta-hxFieldName">Options</legend> -->
+            <!-- <hx-checkbox-control>
+              <input
+                id="chkBeaconInlineStyles"
+                type="checkbox"
+                v-model="isDisabled"
+              />
+              <label for="chkBeaconInlineStyles">
+                <hx-checkbox></hx-checkbox>
+                Apply styles
+              </label>
+            </hx-checkbox-control> -->
+            <hx-text-control>
```

Comment:
### Simplified this demo based on @lalithkarikelli feedback.  Before, the demo provided an option to disable the inline styles.

<img width="789" alt="Screen Shot 2020-06-18 at 10 24 10 AM" src="https://user-images.githubusercontent.com/10120600/85041501-9e9f6380-b14f-11ea-8141-d6b0ce1da034.png">